NMS-13608: Fix several typos

jasper-vandemalle opened the pull request:
### All Contributors

* [x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?
* [x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  If you're not sure how to do this, we can help you get things set up properly.
* [x] Have you made a comment in that issue which points back to this PR?
* [x] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [x] If this is a new or updated feature, is there documentation for the new behavior?
* [x] If this is new code, are there unit and/or integration tests?
* [x] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-13608

mmahacek commented:
@jasper-vandemalle Thanks for contributing!  We've been doing quite a bit to update the docs, and appreciate the work to identify these typos.

I notice your targeted this PR against the `develop` branch, though a number of these files have been updated in the newer `foundation-2021` branch.  Would you be able to merge that into your branch and retarget the PR?
Let us know if you have questions.

mmahacek commented on docs/modules/operation/pages/service-assurance/monitors/MemcachedMonitor.adoc:
```diff
@@ -87,7 +87,7 @@ The following metrics are collected:
 
 | retry
-| Number of attempts to establish the Memcached connnection.
+| Number of attempts to establish the Memcached connection.
```
Will be resolved via https://github.com/OpenNMS/opennms/pull/3624

jasper-vandemalle replied:
I've removed this change from the PR.

mmahacek commented on docs/modules/operation/pages/performance-data-collection/collectors/SnmpCollector.adoc:
```diff
@@ -47,7 +47,7 @@ Configure credentials to access SNMP agents in the Web UI (*Admin>Configure SNMP
 After creating a new datacollection config file, add it to `etc/datacollection-config.xml` for inclusion in polling.
 The `datacollection-config.xml` is reloaded automatically whenever the file is modified.
-The content of dataCollectionGroups in `etc/datacollection/*.xml` files are reloaded each time the Collectd process gathers metics from a node.
+The content of dataCollectionGroups in `etc/datacollection/*.xml` files are reloaded each time the Collectd process gathers metrics from a node.
```
Will be fixed via https://github.com/OpenNMS/opennms/pull/3630 as this file is slated to move to a different folder.

jasper-vandemalle replied:
I've removed this change from the PR.

mmahacek commented on docs/modules/deployment/pages/core/ubuntu/install-core.adoc:
```diff
@@ -18,7 +18,7 @@ NOTE: You can safely ignore the message about conflicting distributions (expecte
 ----
 
-.Installing R package for trending and forcasting (optional)
+.Installing R package for trending and forecasting (optional)
```
```suggestion
.Install R packages for trending and forecasting (optional)
```

jasper-vandemalle replied:
Fixed.

mmahacek commented on docs/modules/deployment/pages/system-scalability/time-series-storage/newts.adoc:
```diff
@@ -90,6 +90,6 @@ describe table samples;
 ----
 
-With this infomration set, {page-component-title} will begin persisting metrics using the Newts engine when restarted.
+With this information set, {page-component-title} will begin persisting metrics using the Newts engine when restarted.
```
Fixed in https://github.com/OpenNMS/opennms/pull/3702

jasper-vandemalle replied:
I've removed this change from the PR.

mmahacek commented on docs/modules/deployment/pages/system-scalability/time-series-storage.adoc:
```diff
@@ -4,7 +4,7 @@
 The following implementations are installed by default:
 
-.Supported Time Series Databasees
+.Supported Time Series Databases
```
Fixed in a later branch

jasper-vandemalle replied:
I kept this change in (for now), as it was still present in `foundation-2021`. Should I remove it as well?

mmahacek replied:
This file will be updated via https://github.com/OpenNMS/opennms/pull/3702

jasper-vandemalle replied:
I've removed this change from the PR.

jasper-vandemalle commented:
> I notice your targeted this PR against the `develop` branch, though a number of these files have been updated in the newer `foundation-2021` branch. Would you be able to merge that into your branch and retarget the PR?

Retargetted my changes to `foundation-2021`.

mmahacek commented on docs/modules/operation/pages/webui/jmx-config-generator/webui.adoc:
```diff
@@ -2,7 +2,7 @@
 
 Complex  JMX  data collection configurations can be generated from a web based tool.
-It collects all available  MBean Attributes  or  Composite Data Attributes  from a  JMX  enabled Java application.
+It collects all available  MBean Attributes or  Composite Data Attributes  from a  JMX  enabled Java application.
```
```suggestion
The JMX Configuration Generator collects all available MBean Attributes or Composite Data Attributes from a JMX enabled Java application.
```

jasper-vandemalle replied:
Fixed.

mmahacek commented on docs/modules/operation/pages/webui/jmx-config-generator/webui.adoc:
```diff
@@ -46,7 +46,7 @@ image::operation/jmx-config-generator/02-webui-mbean-selection.png[]
 To select or deselect all attributes use *Mouse right click -> select/deselect*.
 
-The right panel shows the MBean Attributes with the RRD-specific mapping and lets you select or deselect specific MBean Attriubtes or composite data attributes for the data collection configuration.
+The right panel shows the MBean Attributes with the RRD-specific mapping and lets you select or deselect specific MBean Attributes  or composite data attributes for the data collection configuration.
```
```suggestion
The right panel shows the MBean Attributes with the RRD-specific mapping, and lets you select or deselect specific MBean Attributes or composite data attributes for the data collection configuration.
```

jasper-vandemalle replied:
Fixed.

mmahacek reviewed (changes requested):
After a further review, I notice there are some config XML files that were modified, which would require targeting `release-28.x` branch instead, but you're in luck that is a downstream branch of `foundation` and doesn't require any other rebasing on your part.
Just a couple other items to fix and then I think we can give the go ahead to merge it.

jasper-vandemalle commented:
> After a further review, I notice there are some config XML files that were modified, which would require targeting `release-28.x` branch instead, but you're in luck that is a downstream branch of `foundation` and doesn't require any other rebasing on your part. Just a couple other items to fix and then I think we can give the go ahead to merge it.

Thanks for the thorough review @mmahacek. I've fixed your remarks.

I've also removed some additional double spaces from `docs/modules/operation/pages/webui/jmx-config-generator/webui.adoc`.

mmahacek commented:
Would you mind pulling changes for the following files out of this PR and making a separate PR against the `release-28.x` branch?

- opennms-base-assembly/src/main/filtered/etc/events/EMC-Celerra.events.xml 
- opennms-config/src/test/resources/org/opennms/netmgt/config/eventd/eventconf-speedtest/events/EMC-Celerra.events.xml
- opennms-config/src/test/resources/org/opennms/netmgt/config/eventd/eventconf-speedtest/events/JuniperEvents/vse-mcg/MME5.0_032.0.20A_VSE_eventconf.xml
- opennms-container/minion/Dockerfile

jasper-vandemalle commented:
Created #3709 with those files.

mszoek commented:
These changes look good to me too! Thanks for cleaning these up :+1:
